s3/s3_operations.cpp

```diff
@@ -377,7 +377,7 @@ namespace irods_s3 {
         }
 
         // ********* DEBUG - print L1desc for all
-        if (debug_log_level == LOG_DEBUG) {
+        if (debug_log_level < LOG_DEBUG) {
```

Comment:
you want this to print if debug_log_level is DEBUG or noisier, right?

so...  i think that means this should be `>= LOG_DEBUG`?

Reply:
Normally the debug_log_level will be LOG_DEBUG (7).  We do not want this loop to run at this level.  To turn on more verbose messages the debug_log_level would be set to LOG_NOTICE (5).  At that point we would want to run this loop for debugging purposes.

Reply:
phone call - decided on only printing this loop out if server properties log level is >= this local developer log level.